components/rmrk/Gallery/History.vue

```diff
@@ -244,6 +246,11 @@ export default class History extends mixins(
     return [HistoryEventType.ALL, Interaction.BUY].includes(this.event)
   }
 
+  get disableBlockUrl() {
```

Comment:
proper naming & return type

```suggestion
  get hasDisabledBlockUrl(): boolean {
```